Sources/PackageMetadata/PackageMetadata.swift

```diff
@@ -87,6 +88,7 @@ public struct PackageSearchClient {
         self.registryClient.getPackageVersionMetadata(
             package: package,
             version: version,
+            fileSystem: localFileSystem,
```

Comment:
probably better for PackageSearchClient to take FileSystem in its constructor instead of defaulting to localFileSystem here